api/clients/verification/verification_utils_test.go

```diff
@@ -0,0 +1,141 @@
+
+	kzgConfig       *kzg.KzgConfig
+	kzgVerifier     *verifier.Verifier
+	testRandom      *random.TestRandom
+	srsNumberToLoad uint64
+)
+
+func setup() {
+	log.Println("Setting up suite")
+
+	srsNumberToLoad = 2900
+
+	kzgConfig = &kzg.KzgConfig{
+		G1Path:          "../../../inabox/resources/kzg/g1.point",
+		G2Path:          "../../../inabox/resources/kzg/g2.point",
+		G2PowerOf2Path:  "../../../inabox/resources/kzg/g2.point.powerOf2",
+		CacheDir:        "../../../inabox/resources/kzg/SRSTables",
+		SRSOrder:        3000,
+		SRSNumberToLoad: srsNumberToLoad,
+		NumWorker:       uint64(runtime.GOMAXPROCS(0)),
+		LoadG2Points:    false,
+	}
+
+	testRandom = random.NewTestRandom()
+}
+
+// randomlyModifyBytes picks a random byte from the input array, and increments it
+func randomlyModifyBytes(inputBytes []byte) {
+	indexToModify := testRandom.Intn(len(inputBytes))
+	inputBytes[indexToModify] = inputBytes[indexToModify] + 1
+}
+
+func TestMain(m *testing.M) {
+	setup()
+	result := m.Run()
+	teardown()
+	os.Exit(result)
+}
+
+func teardown() {
+	log.Println("Tearing down")
+}
+
+func TestComputeAndCompareKzgCommitmentSuccess(t *testing.T) {
+	kzgVerifier, err := verifier.NewVerifier(kzgConfig, nil)
+	assert.NotNil(t, kzgVerifier)
+	assert.Nil(t, err)
+
+	commitment, err := GenerateBlobCommitment(kzgVerifier, gettysburgAddressBytes)
+	assert.NotNil(t, commitment)
+	assert.Nil(t, err)
+
+	// make sure the commitment verifies correctly
+	err = GenerateAndCompareBlobCommitment(
+		kzgVerifier,
+		commitment,
+		gettysburgAddressBytes)
+	assert.Nil(t, err)
```

Comment:
This will yield a better error message when checking for errors.
```suggestion
	require.NoError(t, err)
```

Reply:
Done